nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-security/pom.xml

```diff
@@ -124,4 +124,92 @@
             <scope>provided</scope>
         </dependency>
     </dependencies>
+    <profiles>
+        <profile>
+            <!-- Custom profile for Groovy tests -->
+            <id>groovyUnitTest</id>
```

Comment:
Do we need this profile with it already included in the root pom? Or if there something different about this one? Nit pick, existing profile id's naming convention ... groovy-unit-test.